File: Services/Leagues/FliGen.Services.Leagues.Application/Dto/LeagueInformation.cs
namespace FliGen.Services.Leagues.Application.Dto
{
    public class LeagueInformation
    {
        public string Name { get; set; }
        public string Description { get; set; }
        public LeagueType LeagueType { get; set; }
        public int SeasonsCount { get; set; }
        public int ToursCount { get; set; }

        public Season CurrentSeason { get; set; }
    }
}

File: Services/Web/FliGen.Web/ClientApp/src/app/common/leagueInformation.ts
import { LeagueType } from "./leagueType";
import { Season } from "./season";

export class LeagueInformation {
  name: string;
  description: string;
  leagueType: LeagueType;
  seasonsCount: number;
  toursCount: number;
  currentSeason: Season;
}
